exec/java-exec/src/main/java/org/apache/drill/exec/work/metadata/ServerMetaProvider.java

```diff
@@ -121,7 +121,7 @@ public void run() {
       final GetServerMetaResp.Builder respBuilder = GetServerMetaResp.newBuilder();
       try {
         final ServerMeta.Builder metaBuilder = ServerMeta.newBuilder(DEFAULT);
-        PlannerSettings plannerSettings = new PlannerSettings(session.getOptions(), context.getFunctionImplementationRegistry());
+        PlannerSettings plannerSettings = new PlannerSettings(session.getOptions(), context.getFunctionImplementationRegistry(), null);
```

Comment:
After passing only the required things, please don't forget to pass them here.

Reply:
Fixed.